app/models/steps/parallel.rb

```diff
@@ -37,7 +37,7 @@ class Parallel < Approval
       end
     end
 
-    def on_actionable_entry(old_state, event)
+    def on_actionable_entry(_, _)
```

Comment:
Don't know, I'm just cleaning up previous code. As far as I know it's still unused.